src/main/java/network/brightspots/rcv/Main.java

```diff
@@ -83,4 +84,13 @@ public static void main(String[] args) {
 
     System.exit(0);
   }
+
+  public static void logSystemInfo() {
+    Logger.log(Level.INFO, String.format("Launching %s version %s...", APP_NAME, APP_VERSION));
```

Comment:
This line is duplicated; either remove here or just before this function is called.